worldengine/basic_map_operations.py

```diff
@@ -1,9 +1,4 @@
 import math
-import random
-
-
-def random_point(width, height):
```

Comment:
It was only used once. Removing it allowed for another numpy-optimization; so for now this function would have been dead code (and since it is fairly simple anyway, rewriting it would be easy).